packages/components/nodes/responsibleAI/InputModeration/SimplePromptModeration.ts

```diff
@@ -0,0 +1,56 @@
+import { INode, INodeData, INodeParams } from '../../../src/Interface'
+import { getBaseClasses } from '../../../src'
+import { Moderation } from '../ResponsibleAI'
+import { SimplePromptModerationRunner } from './SimplePromptModerationRunner'
+
+class SimplePromptModeration implements INode {
+    label: string
+    name: string
+    version: number
+    description: string
+    type: string
+    icon: string
+    category: string
+    baseClasses: string[]
+    inputs: INodeParams[]
+
+    constructor() {
+        this.label = 'Moderation - Simple Prompt'
+        this.name = 'inputModerationSimple'
+        this.version = 1.0
+        this.type = 'Moderation'
+        this.icon = 'simple_moderation.png'
+        this.category = 'Responsible AI'
+        this.description = 'Detecting and mitigating prompt attacks'
+        this.baseClasses = [this.type, ...getBaseClasses(Moderation)]
+        this.inputs = [
+            {
+                label: 'Deny List',
+                name: 'denyList',
```

Comment:
is it better to be separated by comma? there might be cases where a string literal contains `\n`